test/fixtures/model/datatype.json

```diff
@@ -7,7 +7,16 @@
       "name": "Root",
       "properties": [
         { "name": "bounds", "serialize": "xsi:type", "type": "Rect" },
-        { "name": "otherBounds", "type": "do:Rect", "serialize": "xsi:type", "isMany": true }
+        { "name": "otherBounds",
```

Comment:
Proposal: Revert to original formatting (single line) like most of the other props.

Reply:
I added another property and was not sure about the formatting. If you take a look at [fixtures/properties.json](https://github.com/fra-pa/moddle/blob/feat-multiple-inherited-properties/test/fixtures/model/properties.json), most of the attributes are in a seperate line, but others are defined in one line (top of the file).

It's not consistant and even though I'm not an ideologist about 80 character line limit, I apply this rule for readability and consistancy if reasonable.

[This commit](https://github.com/bpmn-io/moddle/pull/39/commits/596d002ca25e6ddb1eccc116eaad15d8beea61a3) changes the formatting back to one-liners and if you prefere this, we keep it. If you like, I'll change the formatting of all fixtures to one style.